activity/activity.info

```diff
@@ -1,8 +1,7 @@
 [Activity]
 name = Flip
-activity_version = 9
-license = GPLv3+;MIT
-exec = sugar-activity3 FlipActivity.FlipActivity
+activity_version = 10
+license = GPLv3
```

Comment:
sprites.py is MIT licensed.

FlipActivity.py is GPLv3+ licensed; note the "any later version".

Reply:
Thanks for pointing it out, fixed it d379bb72. Happened when I was rebasing and I'd thought I'd made all the needed changes.